packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/validate-bucket-name.js

```diff
@@ -1,8 +1,8 @@
 module.exports = (value) => {
-  let isValid = typeof value === 'string' && value.length >= 3 && value.length <= 63 && /^[a-z0-9.-]*$/.test(value);
+  let isValid = typeof value === 'string' && value.length >= 3 && /^[a-z0-9.-]*$/.test(value);
```

Comment:
Thanks for contributing! 

Unfortunately, S3 has a limitation to the bucket names

> The bucket name can be between 3 and 63 characters long, and can contain only lower-case characters, numbers, periods, and dashes.

Please check out here https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-trail-naming-requirements.html

Reply:
Apologies for not looking into that first, You think the issue should be closed since[ bucket name rules](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html#general-purpose-bucket-names) dont allow it.